File: Payanak/Original/ClientApp/src/app/shared/model/sms/business-card.model.ts
import {ContactModel} from '../contact/contact.model';
import {GroupModel} from '../group.model';
import {TemplateModel} from './template.model';
import {NumberModel} from '../number/number.model';

export class BusinessCardModel {
  id: number;
  createDate: number;
  isBlocked: boolean;
  status: number;
  user: ContactModel;
  group: GroupModel;
  template: TemplateModel;
  number: NumberModel;
  key: string;
}

File: Payanak/Original/ClientModels/BusinessCardModel.cs
namespace Backend.ClientModels{
    public class BusinessCardModel
    {
        public long Id{get;set;}
        public string Key{get;set;}
        public long CreateDate {get;set;}
        public bool IsBlocked{get;set;}
        public short Status{get;set;}
        public ContactModel User{get;set;}
        public GroupModel Group{get;set;}
        public TemplateModel Template{get;set;}
        public NumberModel Number{get;set;}
    }
}